OPHD/UI/PopulationPanel.cpp

```diff
@@ -14,6 +14,39 @@
 using namespace NAS2D;
 
 
+static const auto formatDiff = [](int diff)
+{
+	return ((diff > 0) ? "+" : "") + std::to_string(diff);
+};
+
+static const auto trendIndex = [](int value)
+{
+	return (value >= 0) ? 0 : 1;
+};
+
+static const std::array trend
+{
+	Color{ 0, 185, 0 },
+	Color::Red
+};
+
+
+static const auto moraleIndex = [](int val)
+{
+	return std::clamp(val, 0, 999) / 200;
+};
+
+
+static const std::array moraleStringColor
+{
+	Color::Red,
+	Color::Orange,
+	Color::Yellow,
+	Color::White,
+	Color{ 0, 185, 0 }
+};
```

Comment:
These could be moved into local variables in the function where they are used.

Reply:
I really hate doing that. It unnecessarily bloats the function when they could just be statically declared in the module in which they're used.